widgets/src/map/IChoropleth.js

```diff
@@ -1,15 +1,31 @@
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["../common/Palette", "./us-states"], factory);
+        define(["../common/Palette", "./us-states","./us-counties"], factory);
```

Comment:
FYI - us-counties is _big_ (and "us-states" is _large_) - I will probably move them to load on demand later on...